src/components/charts/useChartGroup.ts

```diff
@@ -225,25 +236,38 @@ export default function useChartGroup(chartGroupId = null): ChartGroupExports {
             showError(err)
         }
     }
-    
+
+    function isValidName(value: string, invalidNames: string[]): string {
+        let err = invalidNames.indexOf(value) === -1 ? '' : 'Duplicate names found'
```

Comment:
invalidNames.indexOf(value) === -1 ? '' : 'Duplicate names to `invalidNames.indexOf(value) === -1 && 'Duplicate names found`